tradehub/authenticated_client.py

```diff
@@ -127,7 +127,7 @@ def sign_message(self,
                      fee: dict = None):     # JS original -> msgs: ConcreteMsg[], options: any = {}
 
         if (not sequence and self.account_sequence_nbr is None) or self.account_nbr is None or self.account_nbr == '0':  # no sequence override, get latest from blockchain
-            self.account_blockchain_dict = self.get_account()
+            self.account_blockchain_dict = self.get_account(self.wallet.address)
```

Comment:
This should probably just be:

self.get_account_details()

It does the same thing here but is internal to this Client.

Reply:
Jup saw this later in the code, just the wrong method called.